CHANGELOG.md

```diff
@@ -51,3 +51,8 @@
 
 - **v0.3.1**
    - Issue 41: Fix logging and add kwarg
+  
+-  **v1.0.0**
+    - Issue 43: Improvement of sensitivity analysis
```

Comment:
"Improvent" --> Major new features to use SA for automatic selection of tuner parameters

This is the breaking change (aside from py37), you can advertise it a bit more :)